eyelid-match-ops/src/primitives/yashe/conf.rs

```diff
@@ -65,8 +101,12 @@ impl YasheConf for FullRes {
 #[cfg(tiny_poly)]
 impl YasheConf for TinyTest {
     /// Limited to the modulus of the underlying `Coeff` type.
-    const T: u64 = 7;
+    const T: u64 = 4;
 
     /// Limited to 1/6 of the modulus, so that the sampled values are valid within 6 sigmas.
-    const DELTA: f64 = 0.9;
+    const KEY_DELTA: f64 = 0.3;
+
+    /// Limited to 1/3 of KEY_DELTA, so that the error is small enough for valid decryption.
+    /// This effectively makes the error zero.
+    const ERROR_DELTA: f64 = 0.1;
```

Comment:
```suggestion
    /// This effectively sets the key randomness to zero, and the key to one, to six sigma probability.
    const KEY_DELTA: f64 = 0.01;

    /// Limited to 1/3 of KEY_DELTA, so that the error is small enough for valid decryption.
    /// This effectively makes the error zero, to six sigma probability.
    const ERROR_DELTA: f64 = 0.01;
```